plat/socionext/synquacer/drivers/scmi/sq_scmi.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2019, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <assert.h>
+
+#include <arch_helpers.h>
+#include <common/debug.h>
+
+#include <drivers/arm/css/scmi.h>
+#include <drivers/arm/css/css_mhu_doorbell.h>
+#include <plat/arm/common/plat_arm.h>
+#include <plat/common/platform.h>
+
+#include "scmi_sq.h"
+
+#include <sq_common.h>
+
+const uint32_t plat_css_core_pos_to_scmi_dmn_id_map[PLATFORM_CORE_COUNT] = {
+	0,   1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11,
+	12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23
+};
+
+#define SQ_SCMI_PAYLOAD_BASE		PLAT_SQ_SCP_COM_SHARED_MEM_BASE
+#define MHU_CPU_INTR_S_SET_OFFSET	0x308
+
+static scmi_channel_plat_info_t sq_scmi_plat_info = {
+		.scmi_mbx_mem = SQ_SCMI_PAYLOAD_BASE,
+		.db_reg_addr = PLAT_CSS_MHU_BASE + MHU_CPU_INTR_S_SET_OFFSET,
```

Comment:
I dont think you need to use any `CSS` macro anymore. This can be `PLAT_SQ_MHU_BASE	`

Reply:
PLAT_CSS_MHU_BASE is used in include/drivers/arm/css/css_mhu_doorbell.h, and I still need
this macro to use generic MHU driver.
So, I submit separate commit to rename PLAT_CSS_MHU_BASE to PLAT_MHUV2_BASE.
I think it is not the best solution, but it is minimal modification without modifying much under plat/arm.